utils/fs.go

```diff
@@ -0,0 +1,18 @@
+package utils
+
+import (
+	"os"
+
+	"github.com/golang/glog"
+)
+
+// Check if the file exists, gracefully handling ENOENT.
```

Comment:
For golang standards the comment should start with the name of the function ... not worth blocking on though.

Reply:
Fixed!